engines/bastion_katello/app/assets/javascripts/bastion_katello/products/discovery/discovery-create.controller.js

```diff
@@ -1,92 +1,119 @@
 /**
  * @ngdoc object
- * @name  Bastion.products.controller:DiscoveryFormController
+ * @name  Bastion.products.controller:DiscoveryCreateController
  *
  * @requires $scope
  * @requires $q
  * @requires CurrentOrganization
  * @requires Product
  * @requires Repository
  * @requires FormUtils
+ * @requires translate
  *
  * @description
  *   Provides the functionality for the repo creation as part of
  *      repository discovery.
  */
-angular.module('Bastion.products').controller('DiscoveryFormController',
-    ['$scope', '$q', 'CurrentOrganization', 'Product', 'Repository', 'GPGKey', 'FormUtils',
-    function ($scope, $q, CurrentOrganization, Product, Repository, GPGKey, FormUtils) {
-
-        function productCreateError(response) {
-            $scope.working = false;
-            $scope.createRepoChoices.creating = false;
-            angular.forEach(response.data.errors, function (errors, field) {
-                if (angular.isUndefined($scope.productForm[field])) {
-                    $scope.productErrorMessages = 'An error occurred while saving the Product: ' + field + ' ' + errors;
-                } else {
-                    $scope.productForm[field].$setValidity('server', false);
-                    $scope.productForm[field].$error.messages = errors;
-                }
-            });
-        }
-
-        function repoCreateError(response) {
-            var currentlyCreating = $scope.currentlyCreating;
-            $scope.currentlyCreating = undefined;
-            $scope.createRepoChoices.creating = false;
-            $scope.productErrorMessages = response.data.errors;
-            currentlyCreating.form.$invalid = true;
-        }
+angular.module('Bastion.products').controller('DiscoveryCreateController',
+    ['$scope', '$q', 'CurrentOrganization', 'Product', 'Repository', 'GPGKey', 'FormUtils', 'translate',
+    function ($scope, $q, CurrentOrganization, Product, Repository, GPGKey, FormUtils, translate) {
+
+        $scope.errorMessages = [];
+        $scope.successMessages = [];
+
+        $scope.table = {
```

Comment:
Happy to correct if you show me how. I tried for hours to get tables working with the data passed in.